packages/mui-material-nextjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/material-nextjs",
-  "version": "6.0.0-alpha.14",
+  "version": "6.0.0-alpha.15",
```

Comment:
Shouldn't this be beta?

Reply:
It has been alpha since we started beta release. I suppose I can change it here.